ports/geckolib/glue.rs

```diff
@@ -328,27 +320,24 @@ pub extern "C" fn Servo_GetComputedValuesForPseudoElement(parent_style: *mut Ser
 }
 
 #[no_mangle]
-pub extern "C" fn Servo_InheritComputedValues(parent_style: *mut ServoComputedValues)
-     -> *mut ServoComputedValues {
-    type Helpers = ArcHelpers<ServoComputedValues, ComputedValues>;
+pub extern "C" fn Servo_InheritComputedValues(parent_style: ServoComputedValuesBorrowed)
+     -> ServoComputedValuesStrong {
     let style = if parent_style.is_null() {
         Arc::new(ComputedValues::initial_values().clone())
     } else {
-        Helpers::with(parent_style, ComputedValues::inherit_from)
+        ComputedValues::with(parent_style, ComputedValues::inherit_from)
     };
-    Helpers::from(style)
+    ComputedValues::from_arc(style)
```

Comment:
that way this could be something like `style.into_gecko()`, or something like it, and make it consume the Arc